packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -407,6 +407,7 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 			!this.heuristicRunnerMicroTaskExists
 		) {
 			this.heuristicRunnerMicroTaskExists = true;
+			// eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
This one feels like it merits a comment to explain why we're floating it. @markfields / @agarwal-navin FYI.